client/src/legacy/CKANResourceLoaderField-entwine.js

```diff
@@ -16,6 +16,7 @@ jQuery.entwine('ss', ($) => {
         defaultEndpoint: schemaData.defaultEndpoint || null,
         description: schemaData.description.html || '',
         value: value ? JSON.parse(value) : undefined,
+        ...schemaData,
```

Comment:
Shouldn't you unpack above the other declarations so that the properties don't get overwritten?